spikes/bullets/BulletManager.cpp

```diff
@@ -0,0 +1,51 @@
+#include "stdafx.h"
+#include "BulletManager.h"
+
+
+BulletManager * BulletManager::instance = 0;
```

Comment:
what will standard for writing pointers be?
BulletManager *name ...or... BulletManager* name